docs/Documentation/Drivers/Classic/OutputFormatting.md

```diff
@@ -59,7 +53,7 @@ _type_ = (for binary output files) data type code. Must be one of:
   - `OUT_TYPE_CHAR` = char
   - `*` = use the default type
```

Comment:
How does the user know what the default type is (since they will need this information to read the files into another program)?